src/InputWithLabel.js

```diff
@@ -0,0 +1,24 @@
+import React, { useRef, useEffect } from 'react';
+
+
+function InputWithLabel(props){
+    const inputRef = useRef();
+useEffect(()=>{
+inputRef.current.focus()
+}, ) 
```

Comment:
`useEffect` should always have a dependency array even if it's blank to avoid unnecessary re-renders.